src/main/java/no/nav/familie/integrasjoner/sftp/SftpClient.kt

```diff
@@ -0,0 +1,37 @@
+package no.nav.familie.integrasjoner.sftp
+
+import com.jcraft.jsch.ChannelSftp
+import com.jcraft.jsch.JSch
+import com.jcraft.jsch.Session
+import no.nav.familie.integrasjoner.config.SftpConfig
+import no.nav.familie.kontrakter.felles.Fil
+import org.springframework.stereotype.Service
+import java.io.ByteArrayInputStream
+
+@Service
+class SftpClient(private val config: SftpConfig) {
+
+    private val jSch = JSch().apply {
+        addIdentity(config.username,
+                    config.privateKeyDecoded,
+                    null,
+                    config.passphrase.toByteArray())
+    }
+
+    fun put(fil: Fil) {
+        var session: Session? = null
+        var channel: ChannelSftp? = null
+        try {
+            session = jSch.getSession(config.username, config.host, config.port).apply { }
```

Comment:
Hva skal denne brukes til? Hvis den skal bli brukt mye så burde man kanskje ha bedre håndtering for å holde en åpen session?

Reply:
Sending av avstemmingsfil en gang i døgnet.

Reply:
👍 Til økonomi? 
Burde endepunkten i stedet hete noe i form av avstemming mot X sånn att man får litt mer context om hva det brukes til?